### PR TITLE
[#41] New option form - UI fixes

### PR DESCRIPTION
**Pull request by FilipKopecky**
The graphics are now fixed. The individual items are now selectable and idividual results fit into the dropdown.
However this PR only fixes the look of the component. The component itself is unusable. 
Form itself can never be submitted due, what I believe is some problem with validation. I have serious doubts that this component is used anywhere. It needs to be reworked in order to be functional. The code for it to work is already there but no one made sure that the individual parts communicate in a way which would be functional. I purposely left commented out code which will be useful upon fixing the component.

I suggest creating a new issue where the intended functionality of this component would be fully described, because now the intended usage is unclear.

Some questions to consider:

1. Should the dropdown have all the functionalities as the 'main' search? e.g. clickable selected options which would redirect user to the definition?
2. Why do the options differ for parent terms and children? Should they differ? 
3. Should there be a tooltip for individual items?

There is no point in continuing with the questions until the component has full description of its intended usage.

tl;dr This PR fixes graphical issues but doesn't fix functionality of the component
resolves: lecbyjak/intelligent-tree-select#41

### CODE REVIEW

**Comment by ledsoft** [comment]
PR closed as agreed, new option creation has been removed from the component at the moment.
